README.md

```diff
@@ -1 +1,11 @@
 # hw01
+
+## Self-introduction
+1. My name is **Rui Tang**. 
+2. I am a *MAPSS* student in _Psychology_.
+1. I am interested in studying individual differences. 
+3. I hope to learn some useful skills in this course!
+
+![Logo](https://cdn.pixabay.com/photo/2018/11/24/13/46/spongebob-3835614_960_720.jpg) 
+
+[My_Page](https://github.com/Ray-RuiTang)
```

Comment:
Excellent job adding this link, though it's maybe a bit redundant to link us back to your github account from the file we are viewing in github ;).